msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1190,6 +1191,27 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 
+### extensions.keyboardShortcuts
+
+The `extensions.keyboardShortcuts` property defines custom keyboard shortcuts or key combinations to run specific actions. For more information, see [Add custom keyboard shortcuts to your Office Add-ins](/office/dev/add-ins/design/keyboard-shortcuts).
+
+| Name | Type | Maximum size | Required | Description |
+|---|---|---|---|---|
+|`requirements`| Object | | | Specifies the scopes, form factors, and Office JavaScript library requirement sets that must be supported on the Office client in order for the custom keyboard shortcuts to take effect. For more information, see [Specify Office Add-in requirements in the unified manifest for Microsoft 365](/office/dev/add-ins/develop/requirements-property-unified-manifest). |
+|`requirements.capabilities`| Array | | | Identifies the requirement sets. |
+|`requirements.capabilities.name`| String | | ✔️ | Identifies the name of the requirement set. |
+|`requirements.capabilities.minVersion`| String | | | Identifies the minimum version for the requirement set. |
+|`requirements.capabilities.maxVersion`| String | | | Identifies the maximum version for the requirement set. |
+|`requirements.scopes`| Array of enums | 4 | | Identifies the scopes in which the add-in can run and defines the Microsoft 365 applications in which the extension can run. <br>Supported value: `workbook` (Excel) |
+|`requirements.formFactors`| Array of enums | 2 | | Identifies the form factors that support the add-in.<br>Supported value: `desktop` |
+|`shortcuts`| Array | 20,000 | ✔️ | Specifies the custom keyboard shortcuts and the action each one performs. |
+|`shortcuts.key`| Object | | ✔️ | Specifies custom key combinations for a single action across supported platforms. |
+|`shortcuts.key.default`| String | 32 | ✔️ | Specifies the default key combination that can be used on all supported platforms if there isn't a specific combination configured for a particular platform. Keyboard shortcuts defined for an add-in must follow the [guidelines for custom key combinations](/office/dev/add-ins/design/keyboard-shortcuts#guidelines-for-custom-key-combinations).  |
+|`shortcuts.key.mac`| String | 32 | | Specifies the key combination of an add-in in Office on Mac. A keyboard shortcut can be any combination of the characters `A-Z`, `a-z`, `0-9`, and the punctuation marks `-`, `_`, and `+`. It must include the name of at least one modifier key (Alt, Command, or Shift) and only one other key. These keys must be joined with a `+` character. For example, `Command+1`. The Shift key can't be used as the only modifier key. It must be combined with either the Alt or Command key. On macOS, the Alt key is mapped to the Option key. |
```

Comment:
Hi @samantharamon,
Guess we can stick to the definition instead of explaining the keys supported as we have already provided the link. If we add, we must adhere to [MSTP guidelines for keys](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts) and the guidelines defined in keyboardshortcuts.

What are your thoughts?

Reply:
@v-ramyas5 That sounds good. If we just stick to the definition, would you recommend adding the link to each entry (not just `shortcuts.key.default`)? I'm leaning towards adding the link to each entry in case the reader skips directly to one of the other key properties. Let me know your thoughts on this.

Reply:
@samantharamon, I prefer adding it as a note instead of adding of it to every field as that would be a repetitive information.

Reply:
Removed the guidelines from each entry and moved it to a note.